docs/sphinx/sections/api/apidocs/libraries/dagster-dbt.rst

```diff
@@ -20,6 +20,10 @@ dagster-dbt
     :prog: dagster-dbt project scaffold
     :nested:
 
+.. click:: dagster_dbt.cli.app:project_app_typer_click_object
+    :prog: dagster-dbt project prepare-for-deployment
+    :nested:
```

Comment:
This is rendering a bit weird, I wonder if we should get rid of `:nested:`, or maybe add some docstring

Reply:
hmm, is there just something up with my local apidoc build artifacts or something? This looks good https://docs.dagster.io/_apidocs/libraries/dagster-dbt

Reply:
I keep getting sphinx errors if i try to set `full` on the `prepare-for-deployment` entry (it works on `scaffold`) no matter what i do with the doc strings, did you get the api doc build to work?

Reply:
yeah I got it to work after I removed all the backticks in `app.py`